sources/web/datalab/templates/terminals.html

```diff
@@ -60,9 +59,8 @@
         1
         2
         3
-        4
-      <span id="dummy-screen-rows" style="">01234567890123456789012345678901234567890123456789012345678901234567890123456789</span>
-      </pre>
+        </pre>
+      <span id="dummy-screen-rows" style="">01234567890123456789012345678901234567890123456789012345678901234567890123456789012</span>
```

Comment:
80 characters wasn't well aligned with the size of the terminal. Unfortunately we have little control over this calculation because of the so many parameters it uses:
- Window dimensions
- Terminal container dimensions
- Line height
- Padding and margin

So I had to adjust it until it fit well with our usage.